php/request-replayer/Dockerfile

```
@@ -0,0 +1,18 @@
+FROM php:7.3
```

What are your thoughts on using a web-server enabled image? Maybe something like this:

```suggestion
FROM php:7.3-apache
```

A few reasons to consider for using a real web server vs the built-in web server: 

- The built-in web server tends to be a bit unreliable (as we've seen in our flaky tests in the CI builds)
- The built-in web server is quite slow, especially compared to a production web server
- The built-in web server has very limited features since it is designed for debugging only - in some of my tests in the past it seems to reject valid requests

If you think that's too big of a change, we can bookmark this for a future PR though. :)

---

Very good idea. I created a task in our backlog to work on this. In the meantime we can get this merged with the old internal server if you are cool with that.